antfs-cli.py

```diff
@@ -243,34 +240,33 @@ def on_transport(self, beacon):
             results = {}
             for typ, files in uploading.items():
                 for filename in files:
-                    #print "upload", typ, filename
+                    # print("upload", typ, filename)
```

Comment:
Feel free to remove the uncommented code here and in the following lines if you like.